config/samples/itz-bund/easyredmine.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: k8s.cloudogu.com/v1
+kind: Dogu
+metadata:
+  name: easyredmine
+  annotations:
+    test: dev
+  labels:
+    dogu.name: easyredmine
+    app: ces
+spec:
+  name: itz-bund/easyredmine
```

Comment:
I think it is best we remove customer references. It is not too hard to create a dogu resource from scratch.